src/sql/sqlops.proposed.d.ts

```diff
@@ -203,4 +203,116 @@ declare module 'sqlops' {
 		 */
 		export function registerModelViewProvider(widgetId: string, handler: (view: ModelView) => void): void;
 	}
+
+	export namespace window {
+		export namespace modelviewdialog {
+			/**
+			 * Create a dialog with the given title
+			 * @param title The title of the dialog, displayed at the top
+			 */
+			export function createDialog(title: string): Dialog;
```

Comment:
the changes in this file look fine to me for now, but we need a little more formal process for adding\removing extensibility APIs.  I'd like to have all extensibility APIs go through a design review process to ensure that changes are consistent, support back-compat, following proper conventions, etc.

Reply:
Good idea, I agree. We should definitely make sure this api design makes sense before finalizing it